buildifier/utils/tempfile.go

```diff
@@ -27,9 +27,12 @@ func (tf *TempFile) WriteTemp(data []byte) (file string, err error) {
 }
 
 // Clean removes all created temporary files
-func (tf *TempFile) Clean() {
+func (tf *TempFile) Clean() error {
 	for _, file := range tf.filenames {
-		os.Remove(file)
+		if err := os.Remove(file); err != nil {
+			return err
```

Comment:
note that this leaves tf.filenames in a precarious state as some are removed and some are not.
Also, if Remove returns os.ErrNotExist -- we can probably drop that as our job is done.

Reply:
I thought about that, but the object is used and cleaned exactly once by the buildifier binary. If it fails to delete some temporary files right before exiting there's nothing meaningful to do anyway.